AP-protocol.md

```diff
@@ -287,9 +287,7 @@ Source Routing Header contains the path to the client, which can be obtained by
 If a drone receives a Message and can forward it to the next hop, it also sends an Ack to the client.
 
 ```rust
-pub struct Ack(AckInner);
-
-struct AckInner {
+struct Ack {
```

Comment:
This is fine but maybe it should be in another pull request.
At least it should be in another commit.

And same for the pub in the L336

Reply:
Ok